packages/api/src/utilities/test/batchExecute.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-console */
+
+const runBatch = ({
+  count = 0,
+  createEntry,
+  execute,
+  numberOfEntries,
+  numberOfentriesEachBatch,
+}) => {
+  if (count >= numberOfEntries) {
+    return Promise.resolve()
+  }
+
+  const numberOfBatchEntries = Math.min(
+    numberOfEntries - count,
+    numberOfentriesEachBatch
+  )
+
+  const batchIdentifier = `${count}.batch`
+  const executeIdentifier = `${batchIdentifier}.execute`
+
+  const batchData = []
+  /* eslint-disable no-plusplus */
+  for (let index = 0; index < numberOfBatchEntries; index++) {
+    batchData[index] = createEntry(count + index)
+  }
+  /* eslint-enabke no-plusplus */
```

Comment:
doesn't really matter but: enabke => enable

(and maybe using `i += 1` should be preferred to disabling the lint rule)